testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -2219,8 +2218,8 @@ def test_purge_and_view_compaction(params_from_base_test_setup, sg_conf_name):
     xattrs_enabled = params_from_base_test_setup['xattrs_enabled']
 
     # This test should only run when using xattr meta storage
-    if not xattrs_enabled:
-        pytest.skip('XATTR tests require --xattrs flag')
+    if not xattrs_enabled or mode == "di":
+        pytest.skip('This test skipped XATTR tests require --xattrs flag or This test cannot run in DI mode')
```

Comment:
The message can be changed to "Either --xattrs flag is not provided or test is not applicable for di mode"

Reply:
done